DependencyInjection/FOSRestExtension.php

```diff
@@ -256,7 +259,11 @@ private function loadView(array $config, XmlFileLoader $loader, ContainerBuilder
             $container->getDefinition('fos_rest.mime_type_listener')->replaceArgument(0, $config['view']['mime_types']['formats']);
         }
 
-        if ($config['view']['view_response_listener']['enabled']) {
+        if ($this->isConfigEnabled($container, $config['view']['view_response_listener'])) {
+            if (!class_exists(SensioFrameworkExtraBundle::class)) {
+                throw new LogicException('To use the view response listener, the "sensio/framework-extra-bundle" package is required.');
```

Comment:
I tried your branch in my project where I don't use `sensio/framework-extra-bundle` but the view response listener.

After removing this new check everything works like expected.

Reply:
This might've been a bit pedantic because the `#[View]` attribute extends the `sensio/framework-extra-bundle` `#[Template]` attribute and that hierarchy was giving me some headaches.  But, with https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2401 included, anything that I had to disable because of that should be able to be turned back on.